VFPC/analyzeFP.cpp

```diff
@@ -222,7 +222,10 @@ bool CVFPCPlugin::IsDestinationMatch(const string& destination, const json& rule
 		return true; // No destination restriction
 	}
 	for (const auto& dest : rule["destinations"]) {
-		if (destination.find(dest.get<string>()) != string::npos) {
+		if (destination.find(dest.get<string>()) != string::npos && dest.get<string>().find("*") == string::npos) { // Check if wildcard is present
```

Comment:
This second check seems a bit superfluous? If the string contains ** wildcards it's never going to match anyway